DiscordBot/Commands/EventCommands.cs

```diff
@@ -381,18 +384,32 @@ private async Task CreateSignupSheet(CommandContext context, int eventKey)
                 return;
             }
 
-            await SendSignupMessage(context, eventKey);
+            try
+            {
+                var signupChannel = context.Guild.Channels
+                    .First(channel => channel.Value.Name == SignupChannelName)
```

Comment:
Minor but you could probably use `FirstOrDefault` here instead and avoid the try/catch (instead check for null). Makes the code less nested and a bit easier to read I think 🙂

Reply:
Done!